install_cni_test.go

```diff
@@ -0,0 +1,135 @@
+package main_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"os"
+	"os/exec"
+	"io/ioutil"
+)
+
+var script = "k8s-install/scripts/install-cni.sh"
+
+var defaultEnvVars = map[string]string {
+	"KUBERNETES_SERVICE_HOST": "127.0.0.1",
+	"KUBERNETES_SERVICE_PORT": "8080",
+	"KUBERNETES_NODE_NAME": "k8s-node-01",
```

Comment:
I think  KUBERNETES_NODE_NAME should be removed since it is not used in the default template.